pkg/azure/plugin.go

```diff
@@ -864,3 +813,41 @@ func Setup(port int, orchestratorServerAddr string) *azurePluginServer {
 	}()
 	return azureServer
 }
+
+// Add an existing Azure resource to a paraglider deployment
+func (s *azurePluginServer) AttachResource(ctx context.Context, attachResourceReq *paragliderpb.AttachResourceRequest) (*paragliderpb.AttachResourceResponse, error) {
```

Comment:
Nit: I think the `Setup()` function should be last in the file

Reply:
Done